libs/pilight/protocols/433.92/eurodomest_switch.c

```diff
@@ -0,0 +1,336 @@
+/*
+	Copyright (C) 2015 woutput
+
+	This file is part of pilight.
+
+	pilight is free software: you can redistribute it and/or modify it under the
+	terms of the GNU General Public License as published by the Free Software
+	Foundation, either version 3 of the License, or (at your option) any later
+	version.
+
+	pilight is distributed in the hope that it will be useful, but WITHOUT ANY
+	WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+	A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with pilight. If not, see	<http://www.gnu.org/licenses/>
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+
+#include "../../core/pilight.h"
+#include "../../core/common.h"
+#include "../../core/dso.h"
+#include "../../core/log.h"
+#include "../protocol.h"
+#include "../../core/binary.h"
+#include "../../core/gc.h"
+#include "eurodomest_switch.h"
+
+#define PULSE_MULTIPLIER	3
+#define MIN_PULSE_LENGTH	142
+#define MAX_PULSE_LENGTH	426
+#define AVG_PULSE_LENGTH	284
```

Comment:
Well, yes and no:
Yes: see the graphs in the second sheet of [this attachment] (http://forum.pilight.org/attachment.php?aid=595) from [the referred thread] (http://forum.pilight.org/Thread-Eurodomest)
No: but I suppose there is no disadvantage in accepting a wider range? Therefore I thought: the wider, the more robust.